src/map/Map.js

```diff
@@ -611,6 +612,8 @@ L.Map = L.Evented.extend({
 		if (this.options.trackResize) {
 			L.DomEvent[onOff](window, 'resize', this._onResize, this);
 		}
+
+		this[onOff]('moveend', this._onMoveEnd);

```

Comment:
humm, but the `_leaflet_pos` value changes at every call, no?
And my first implementation was keeping the `if (L.Browser.any3d && this.options.translate3dLimit)` here, but then I thought that this was a trap for people wanting to override `L.Map` to have `_onMoveEnd` sometimes registered sometimes not. One can argue that `_xxx` means private method and we don't care about people overriding but…. ;)

Thoughts?